swagger/oauth.yaml

```diff
@@ -9,6 +9,11 @@ endpoints:
       operationId: request_access_token
       tags:
         - oauth2
+      description: |
+        To request an access token, the client obtains authorization from the
+        resource owner.  The authorization is expressed in the form of an
+        authorization grant, which the client uses to request the access
+        token. See https://tools.ietf.org/html/rfc6749#section-4 for more detail.
```

Comment:
This is one workflow, and it's the one used by IDAs, but there are others.  The mobile apps, for instance, use resource owner credentials (username / password) to request an access token, rather than an authorization grant.  If this is intended to be IDA specific, the current wording should be sufficient, but if mobile apps should also be going through the API gateway, we'll want to make this description more robust.